src/frontend/sharedComponents/DeviceNameWithIcon.tsx

```diff
@@ -52,6 +59,14 @@ export const DeviceNameWithIcon = ({
             {formatMessage(m.thisDevice)}
           </Text>
         )}
+        {isConnected && (
```

Comment:
This looks backwards to me. Should this instead be `!isConnected`?

```suggestion
        {!isConnected && (
```

Reply:
Oh, my mistake. I believe I forgot to make that change after testing it earlier.